.github/workflows/playwright.yml

```diff
@@ -9,6 +9,7 @@ on:
       - '.github/workflows/sync-default.yml'
       - '.github/tests/*.bats'
       - 'private/scripts/**'
+      - 'devops/**'
```

Comment:
Making a change to this file explicitly does _not_ exclude this PR from running these tests because we just made a change to `playwright.yml` which triggers the workflow. `¯\_(ツ)_/¯`